src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -210,22 +220,32 @@ def __init__(self, target: str, options: ChannelArgumentType,
           interceptors: An optional list of interceptors that would be used for
             intercepting any RPC executed with that channel.
         """
-        if interceptors is None:
-            self._unary_unary_interceptors = None
-        else:
-            self._unary_unary_interceptors = list(
-                filter(
-                    lambda interceptor: isinstance(interceptor,
-                                                   UnaryUnaryClientInterceptor),
-                    interceptors))
+        self._unary_unary_interceptors = []
+        self._unary_stream_interceptors = []
+
+        if interceptors:
+            attrs_and_interceptor_classes = [
```

Comment:
Nit: This should be immutable. A tuple is probably a better option. If you're willing to use `getattr`, we can even lift this to a class-level constant.